src/prefect/server/services/base.py

```diff
@@ -1,12 +1,298 @@
+from __future__ import annotations
+
+import abc
+import asyncio
+import inspect
+import signal
 from abc import ABC, abstractmethod
-from typing import NoReturn
+from operator import methodcaller
+from types import ModuleType
+from typing import Any, List, NoReturn, Optional, Sequence, overload
+
+import anyio
+from typing_extensions import Self
+
+import prefect.settings
+from prefect.logging.loggers import get_logger
+from prefect.settings import PREFECT_API_LOG_RETRYABLE_ERRORS
+from prefect.types import DateTime
+from prefect.utilities.processutils import (
+    _register_signal,  # type: ignore[reportPrivateUsage]
+)
+
+
+def _known_service_modules() -> list[ModuleType]:
```

Comment:
This is that spot where we still have to explicitly expose all the service modules (I moved this from the CLI module).  This essentially serves as a "registry" of the services